sdk/relay/Azure.ResourceManager.Relay/tests/Tests/RelayNamespaceTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Relay.Models;
+using Azure.ResourceManager.Relay.Tests.Helpers;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Relay.Tests.Tests
+{
+    public class RelayNamespaceTests: RelayTestBase
+    {
+        private ResourceGroupResource _resourceGroup;
+
+        private RelayNamespaceResource _relayNamespace;
+
+        public RelayNamespaceTests(bool async) : base(async)
+        {
+        }
+
+        [SetUp]
+        public async Task SetUp()
+        {
+            _resourceGroup = await CreateResourceGroupAsync();
+        }
+
+        [Test]
+        [RecordedTest]
+        public async Task RelayNamespaceandCreateOrUpdateTest()
```

Comment:
Could you please double check this test? The CI complained the expected request record isn't found.